libc/atexit.c

```diff
@@ -16,3 +16,10 @@
 #undef OE_NEED_STDC_NAMES
 #undef __UNDEF_OE_NEED_STDC_NAMES
 #endif
+
+extern int oe_cxa_atexit(void (*func)(void*), void* arg, void* dso_handle);
+
+int __cxa_atexit(void (*func)(void*), void* arg, void* dso_handle)
```

Comment:
I'm curious why wouldn't this conflict with alternate libc implementations?

Reply:
Because this is the libc implementation, which sgx-lkl doesn't use.